frontend/lib/ifixit-api/international-buy-box.ts

```diff
@@ -0,0 +1,33 @@
+import { IFixitAPIClient } from '@ifixit/ifixit-api-client';
+import { invariant } from '@ifixit/helpers';
+
+export type BuyBoxVariant = {
+   sku: string;
+   optionid: number;
+   productURL: string;
+   price: string;
+   currency: string;
+   avail: number;
+   salesChannel: string;
+   gaSku: string;
+};
+
+export type BuyBoxResponse = {
+   countryCode: string;
+   storeid: string;
+   storeName: string;
+   storeUrl: string;
+   salesChannel: string;
+   // Keyed by optionid
+   products: Record<string, BuyBoxVariant>;
+};
+
+export function getBuyBoxForProduct(
+   client: IFixitAPIClient,
+   productcode: string
+): Promise<BuyBoxResponse | null> {
+   // TODO pull country override from browser url
```

Comment:
Should this be done? Not sure how we'll QA without this.

Reply:
I'm pretty sure setting `country=` is just a shortcut to changing your preferred store. When we "geo-locate" you, we use the preferred store if it's set.